app/api/sheet/route.ts

```diff
@@ -61,6 +61,14 @@ const response = `<!doctype html>
 	.replace(
 		"{{INKEEP_ORGANIZATION_ID}}",
 		process.env.INKEEP_ORGANIZATION_ID as string,
+	)
+	.replace(
+		"{{INKEEP_ORGANIZATION_DISPLAY_NAME}}",
+		process.env.INKEEP_ORGANIZATION_DISPLAY_NAME as string,
```

Comment:
if INKEEP_ORGANIZATION_DISPLAY_NAME is null or not provided use "Your Organization Name" instead.